nesta_ds_utils/file_ops.py

```diff
@@ -1,53 +1,151 @@
-from io import BytesIO
-from typing import Union
+import io
+from typing import Union, List
 from pathlib import Path
 from xmlrpc.client import Boolean
```

Comment:
Is this an accidental auto import? Looks like you're using it for a type hint below, but I think `bool` is fine.

Reply:
Yes good point, I will change this.